DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/explicit_data_movement/src/explicit_data_movement.cpp

```diff
@@ -123,8 +123,8 @@ double SubmitExplicitKernel(queue& q, std::vector<T>& in,
 
   // free the device memory
   // note that these are calls to sycl::free()
-  sycl::free(in_ptr, q);
-  sycl::free(out_ptr, q);
+  free(in_ptr, q);
+  free(out_ptr, q);
```

Comment:
Nice to see that the code sample coding rules are collapsing by themselves here without needing further discussion... ;-)

Reply:
> > @pmpeter1 done. My compromise was to remove the prefix but add a comment saying this uses sycl::free
> 
> Welcome to Hell!

Can you elaborate? I am guessing you are for keeping the `sycl` namespace here? Do you think it would be better if we dropped `using namespace sycl` and used the `sycl::` prefix everywhere? (These are not rhetorical).

Reply:
The extra comment note is unnecessary.

Reply:
I think that once we need to have comments to explain in which namespace is a symbol, it is better to have the namespace explicitly.

Reply:
I agree. That is why I started with the namespace.